packages/components/multiselect/src/Multiselect.tsx

```diff
@@ -17,6 +17,20 @@ import FocusLock from 'react-focus-lock';
 import type { MultiselectSearchProps as SearchProps } from './MultiselectSearch';
 import { MultiselectSearch } from './MultiselectSearch';
 
+type ClearButtonProps = {
+  /**
+   * Aria label for the clear button
+   * @default 'Clear selection'
+   */
+  ariaLabel?: string;
```

Comment:
Both properties are labeled as optional but there is no fallback for either of them, only for the whole object. If you overwrite only one, the other will be empty. Is that intended?